libs/bootstrap-desktop.sh

```diff
@@ -12,6 +12,7 @@ else
   else
     APPSERVICES_PLATFORM_DIR="$(pwd)/libs/desktop/linux-x86-64"
   fi
+  if [ -z "${ANDROID_NDK_HOME}" ]; then export ANDROID_NDK_HOME="${ANDROID_NDK_ROOT}"; fi
```

Comment:
I don't think we should do that export here. I would instead do it in `libs/verify-android-environment.sh`.

Reply:
IIUC `libs/verify-android-environment.sh` doesn't get `source`d so we can't `export` from there, but it's probably sufficient to add the instructions to the howto and to add a check in `libs/verify-android-environment.sh` that errors out with a helpful message.